### PR TITLE
Add introduction to seismogram

### PR DESCRIPTION
**Pull request by core-man**
#6

### CODE REVIEW

**Comment by seisman** [comment]
https://seismo-learn.org/sitepreview/seismo-learn/seismology101/add-introduction-to-seismic-data/seismogram/ 

This page is empty. Do you want to move the introduction part to the index.rst file, so they will be shown in this page?

**Comment by core-man** [comment]
> https://seismo-learn.org/sitepreview/seismo-learn/seismology101/add-introduction-to-seismic-data/seismogram/
> 
> This page is empty. Do you want to move the introduction part to the index.rst file, so they will be shown in this page?

We can decide whether to do it later when we finish the other sections. Sometimes, some texts may be put in either the index or introduction parts. The introduction part is really hard~ I'd like to come back later to polish it along with index.

File: source/seismogram/index.rst [hunk 1]
```diff
@@ -0,0 +1,9 @@
+地震图
+======
+
+地震图 (seismogram) 是指地震仪记录到的带有记时信息的地面质点运动（位移、速度、加速度、转动等），
```
Comment (seisman) [inline source/seismogram/index.rst:4]: 计时 or 记时？
```suggestion
地震图 (seismogram) 是指地震仪记录到的带有计时信息的地面质点运动（位移、速度、加速度、转动等），
```

File: source/seismogram/index.rst [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+也称为地震波形 (seismic waveform)。其一般有三个正交分量，如南北、东西和垂直分量，
+因此地震图的本质是三维时间序列。
```
Comment (seisman) [inline source/seismogram/index.rst:6]: 感觉“三维”一词用的不太准确。

File: source/seismogram/index.rst [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+
+地震图与元数据 (metadata) 共同构成了完备的地震学观测数据。元数据包含与地震图相关的重要信息，
+如台站位置、仪器响应、震源位置、震源机制等。
```
Comment (seisman) [inline source/seismogram/index.rst:9]: I don't think earthquake location and earthquake focal mechanisms belong to metadata.

File: source/seismogram/index.rst (hunk 1 above)
Comment (core-man) [reply source/seismogram/index.rst:4]: Yes, you are right. It is 计时. Niubitlity~

File: source/seismogram/index.rst (hunk 2 above)
Comment (core-man) [reply source/seismogram/index.rst:6]: 三分量？

File: source/seismogram/index.rst (hunk 3 above)
Comment (core-man) [reply source/seismogram/index.rst:9]: How do you define metadata?

File: source/seismogram/index.rst (hunk 2 above)
Comment (seisman) [reply source/seismogram/index.rst:6]: "三个正交方向上的时间序列"？

File: source/seismogram/index.rst (hunk 3 above)
Comment (seisman) [reply source/seismogram/index.rst:9]: metadata is the information of stations or instruments, not earthquakes.

A seismogram can be full of noise without any earthquake information

File: source/seismogram/index.rst (hunk 2 above)
Comment (core-man) [reply source/seismogram/index.rst:6]: how about only 时间序列？

Comment (seisman) [reply source/seismogram/index.rst:6]: OK to me

File: source/seismogram/index.rst (hunk 3 above)
Comment (core-man) [reply source/seismogram/index.rst:9]: Yes, I agree. metadata is the "data" that are not the real data, while event information and seismic structures are the results we invert from data along with metadata.

Comment (seisman) [reply source/seismogram/index.rst:9]: Yes

**Review by seisman — APPROVED** [review]

**Pull request merged**